### ISSUE TITLE
Cut 0.29.0 release

### ISSUE THREAD

**Issue by maxgio92** [falcosecurity/falco#1668]
## Scheduled to happen: 2021-06-09
We can make the release happen as the features are merged.

## Release Steps
Process is described in [this document](https://github.com/falcosecurity/falco/blob/master/RELEASE.md)

## Release Blocking PRs

- [x] [libs #48](https://github.com/falcosecurity/libs/pull/48)
- [x] #1669
- [x] [libs #50](https://github.com/falcosecurity/libs/pull/50)
- [x] #1669
- [x] [test-infra #434](https://github.com/falcosecurity/test-infra/pull/434)

## Action Items

- [x] Pre-release: [Prebuilt drivers publish](https://prow.falco.org/?repo=falcosecurity%2Ftest-infra&type=postsubmit)
- [x] Pre-release: Milestones
- [x] Pre-release: Changelog #1677 #1682
- [x] Release: Tag
- [x] Release: GH Release
- [x] Docs: Meeting-notes https://github.com/falcosecurity/community/pull/140
- [x] Announcements: Blog [falco-website #457](https://github.com/falcosecurity/falco-website/pull/457)
- [x] Announcements: Send email to cncf-falco-dev mailing list
- [x] Announcements: Falco Slack channel
- [x] Website [snapshot](https://github.com/falcosecurity/falco-website/blob/master/release.md#create-a-new-website-snapshot)

## Open Questions
/milestone 0.29.0
/assign @maxgio92 @leogr @jonahjon 
/kind documentation

**Comment by leogr** [comment]
This is the driver upgrade we need to merge before releasing 0.29.0: https://github.com/falcosecurity/falco/pull/1669

It includes the fix for the packages build [issue](https://github.com/falcosecurity/falco/issues/1653#issuecomment-850438225) we had recently.

**Comment by maxgio92** [comment]
Thanks @leogr!

I think we should move [the other open PRs](https://github.com/falcosecurity/falco/pulls?q=is%3Apr+is%3Aopen) (except the #1669) to a next milestone. WDYT?

**Comment by maxgio92** [comment]
Hello @falcosecurity/falco-maintainers, I think we should postpone this release in a couple of days as we need new (version [41b1e](https://github.com/falcosecurity/libs/pull/48)) prebuilt drivers ready but we have to wait for previous and blocking updates:
  - [May 27th updates](https://github.com/falcosecurity/test-infra/pull/411)
  - [June 2nd updates](https://github.com/falcosecurity/test-infra/pull/417)
  - [June 6th updates](https://github.com/falcosecurity/test-infra/pull/419)

I think that this is mainly due to the [test-infra #421 issue](https://github.com/falcosecurity/test-infra/issues/421) for which the [test-infra fix PR #422](https://github.com/falcosecurity/test-infra/pull/422) should be ready to be approved if you like it.

Then (not needed now), [this test-infra PR #410](https://github.com/falcosecurity/test-infra/pull/410) should then help to speed up prebuilt drivers building by splitting the most time-intensive jobs for Ubuntu distro.

What do you think?

**Comment by jonahjon** [comment]
Anything to speed up the drivers build job would be preferable.

**Comment by jonahjon** [comment]
Like the idea @maxgio92

**Comment by maxgio92** [comment]
Hello @falcosecurity/falco-maintainers, as discussed with @leogr we'd wait to publish the [latest version](https://github.com/falcosecurity/libs/commit/41b1e96d4a275a02919374e4a1d45186d3be9959) of the prebuilt drivers until #1669 is merged.
Doing so we prevent from being stuck in case that candidate driver version for 0.29.0 won't be the confirmed.
WDYT?

**Comment by maxgio92** [comment]
Update:
[libs #50](https://github.com/falcosecurity/libs/pull/50) has been merged! 🥳

**Comment by maxgio92** [comment]
Update:
Waiting for #1675 to be completed so that we'll ship `userfaultd` support with 0.29.0 🚀

**Comment by leodido** [comment]
In the meantime, I send https://github.com/falcosecurity/test-infra/pull/434 to create the Falco prebuilt drivers for 0.29.0.

**Comment by maxgio92** [comment]
Update:

As discussed we decided to go on with the release as the prebuilt drivers are ready 🚀

**Comment by leogr** [comment]
Update:
Since the only precondition left is the building the drivers, I have moved forward.
So I have moved the PRs not yet completed to a new minor milestone.

**Comment by maxgio92** [comment]
Update:
Prepared the [blog release announcement](https://github.com/falcosecurity/falco-website/pull/457)

**Comment by maxgio92** [comment]
Update:
A changelog update has been proposed #1682

**Comment by maxgio92** [comment]
Update:
Prebuilt drivers have been published [here](https://download.falco.org/?prefix=driver/17f5df52a7d9ed6bb12d3b1768460def8439936d/).

**Comment by maxgio92** [comment]
I think we're ready to tag!

**Comment by leogr** [comment]
```shell
❯ git push origin 0.29.0
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
To https://github.com/falcosecurity/falco
 * [new tag]           0.29.0 -> 0.29.0
```
 :rocket:

The CI started :point_down: 
https://app.circleci.com/pipelines/github/falcosecurity/falco/1574/workflows/55d57de1-a8a8-4f3e-a79c-ee818dad860f

**Comment by leogr** [comment]
https://github.com/falcosecurity/falco/releases/tag/0.29.0 :partying_face:

**Comment by leogr** [comment]
Website snapshot for 0.28.0 :point_right: https://v0-28.falco.org/

**Comment by maxgio92** [comment]
Thanks @leogr!

**Comment by maxgio92** [comment]
I think we can close this :)
